src/main/java/org/folio/eusage/reports/api/EusageReportsApi.java

```diff
@@ -1232,6 +1256,10 @@ static String getRanges(JsonArray ar, String startProp, String endProp) {
             + "invoiceNumber text"
             + ")")
         .execute().mapEmpty());
+    future = future.compose(x -> pool
+        .query("CREATE INDEX IF NOT EXISTS agreement_entries_agreementId ON "
+            + agreementEntriesTable(pool) + " USING btree(agreementId)")
+        .execute().mapEmpty());
```

Comment:
The repeated wrapping code could be extracted into a helper method that takes a varargs of SQL strings and executes them.

Reply:
Right. Will do next time, indices are added.